README.md

```diff
@@ -39,19 +39,29 @@ To run the internet_identity canisters, proceed as follows after cloning the rep
 ```bash
 npm install
 dfx start [--clean] [--background]
+```
+
+In a different terminal, run the following command to install the Internet Identity canister:
+
+```bash
 II_ENV=development dfx deploy --no-wallet --argument '(null)'
 ```
 
 Then the canister can be used as
 
 ```bash
-dfx canister call internet_identity register '(123, "test", vec {1; 2; 3}, null)'
+$ dfx canister call internet_identity init_salt
+()
+$ echo $?
+0
 ```
 
-To open the front-end, you can run the following and open the URL.
+See `dfx canister call --help` and [the documentation](https://sdk.dfinity.org/docs/developers-guide/cli-reference/dfx-canister.html#_examples) for more information.
+
+The `dfx` executable can proxy queries to the canister. To view it, run the following:
 
 ```bash
-echo "http://localhost:8000?canisterId=$(dfx canister id internet_identity)"
+open "http://localhost:8000?canisterId=$(dfx canister id internet_identity)"
```

Comment:
`open` is a MacOS specific command, that I'd rather not include here. On Linux it's `xdg-open`. I prefer the previous instruction to just open the URL however people do that on their machine.